test/resample.jl

```diff
@@ -94,16 +94,16 @@ end
     idxLower  = round(Int, yLen/3)
     idxUpper  = idxLower*2
     yDelta    = abs.(f32_y[idxLower:idxUpper].-yy[idxLower:idxUpper])
-    @test all(map(delta -> abs(delta) < 0.005, yDelta))
+    @test all(map(delta -> abs(delta) < 0.0055, yDelta))
```

Comment:
I haven't looked into why this tolerance needs to be increased. Probably just bad luck because with different signal length, a slightly different excerpt is considered, but I should double-check that's all there is to it.

Reply:
Ah, this is hilarious: The test signal `y` is obtained by upsampling (using `resample`) an input `x` by `ratio = 3.141592653589793`, but then the reference signal `yy` is computed based on a time-step reduced by the factor `length(y)/length(u)` (where `length(u)==1000`), which only approximates the `ratio`. And this PR changes `length(y)` from 3141 to 3142 (ref. https://github.com/JuliaDSP/DSP.jl/pull/596#issuecomment-2505660227 on whether that is good) making that approximation worse. Arguably, it would be more reasonable to use `ratio` for the reference signal as well, which would allow this bound to be reduced to `0.00025` and make the impact of this PR on that error insignificant.

I'll open a separate PR to improve the test and then after a rebase, adjusting the test here should be unnecessary.